component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/chain/AutoChunkProcessor.java

```diff
@@ -55,6 +58,7 @@ public void flush(final OutputFactory outs) {
 
     @Override
     public void stop() {
+        flush(remains);
```

Comment:
It looks like we are doing something wrong with this OutputFactory parameter being saved from the parameters.
Can we invoke flush before "stop()" on the client side?